asab/application.py

```diff
@@ -607,8 +607,11 @@ def _set_housekeeping_time_from_config(self):
 	def _on_housekeeping_tick(self, message_type):
 		"""Check if it's time for 'Application.housekeeping!'.
 		If so, publish the message and set housekeeping time for the next day.
+		There is a time limit that prevents from the accidental publishing.
 		"""
-		if datetime.datetime.now(datetime.timezone.utc) > self.NextHousekeeping:
+		now = datetime.datetime.now(datetime.timezone.utc)
+		time_limit = self.NextHousekeeping + datetime.timedelta(hours=2)
```

Comment:
i suggest making this configurable. or at least please extract this value into a member variable.